File: alyak-api-server/src/main/java/com/github/seungjae97/alyak/alyakapiserver/dto/UserDto.java
package com.github.seungjae97.alyak.alyakapiserver.dto;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
public class UserDto {
    private int idx;
    private String email;
    private String password;
    private String name;
    private enum Gender { M, F };
    private String resident_registration_number;
    private String phone_number;
}

File: application_project/app/src/main/java/com/alyak/detector/data/dto/user/UserDto.kt
package com.alyak.detector.data.dto.user

import com.kakao.sdk.user.model.Gender

data class UserDto(
    val email: String,
    val password: String,
    val name: String,
    val phoneNumber: String,
    val gender: Gender,
    val residentNumber: String
)